weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx

```diff
@@ -19,11 +20,19 @@ export const CallSummary: React.FC<{
   call: CallSchema;
 }> = ({call}) => {
   const span = call.rawSpan;
-  const attributes = _.fromPairs(
-    Object.entries(span.attributes ?? {}).filter(
-      ([k, a]) => !k.startsWith('_') && a != null
-    )
+
+  // Process attributes, only filtering out null values and keys starting with '_'
```

Comment:
was this just refactored for clarity?

Reply:
Mostly cause I ran over it a few times - did a couple of implementations that didn't work, but when I backed them out I was able to read this a bit better.